### ISSUE TITLE
SIGSEGV in Unit Test (occasional): multiply_04_n

### ISSUE THREAD

**Issue by myungjoo** [nnstreamer/nntrainer#2459]
In GBS/i586 build w/ unit_test

```
...
  [ 1102s] [ RUN      ] nntrainer_Tensor.multiply_01_p
  [ 1102s] [       OK ] nntrainer_Tensor.multiply_01_p (0 ms)
  [ 1102s] [ RUN      ] nntrainer_Tensor.multiply_02_p
  [ 1102s] [       OK ] nntrainer_Tensor.multiply_02_p (0 ms)
  [ 1102s] [ RUN      ] nntrainer_Tensor.multiply_03_n
  [ 1102s] [       OK ] nntrainer_Tensor.multiply_03_n (0 ms)
  [ 1102s] [ RUN      ] nntrainer_Tensor.multiply_04_n
  [ 1102s] ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
  [ 1102s] 
...
  [ 1111s] 
  [ 1111s] Summary of Failures:
  [ 1111s] 
  [ 1111s]  9/41 unittest_nntrainer_tensor_v2     FAIL             0.26s   killed by signal 11 SIGSEGV
  [ 1111s] 
```

This was in github-action:  https://github.com/nnstreamer/nnstreamer/actions/runs/7751592324/job/21139738200?pr=4399

**Comment by djeong20** [comment]
this should be resolved by #2453. is this still the case?

**Comment by myungjoo** [comment]
> this should be resolved by #2453. is this still the case?

Let's see: https://github.com/nnstreamer/nnstreamer/actions/runs/7751592324/job/21894777282?pr=4399

**Comment by myungjoo** [comment]
It is confirmed to be fixed.
